app/lib/widgets/wallets/add_edit_contact.dart

```diff
@@ -39,6 +39,7 @@ class _AddEditContactState extends State<AddEditContact> {
   bool saveLoading = false;
   String? nameError;
   String? addressError;
+  String _selectedChainType = ChainType.TFChain.name;

```

Comment:
please use the ChainType, and the default should be the widget type
```suggestion
  ChainType _selectedChainType = widget.chainType;

```